<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IRC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_button">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>70</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>217</width>
     <height>69</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Create_rooms_2">
    <item row="1" column="0">
     <widget class="QLineEdit" name="channel_input"/>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="title_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Create channel:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="channel_create_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>581</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Connect_2">
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Manage Port:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Info Port:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="port_manage_input">
      <property name="text">
       <string>25500</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>IP Adress:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="port_info_input">
      <property name="text">
       <string>25501</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="ip_input">
      <property name="text">
       <string>192.168.2.6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>User Name:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="user_input"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>211</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Rooms_2">
    <item row="0" column="0">
     <widget class="QLabel" name="title_5">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Channels:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="channel_list"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>184</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="Messages_2">
    <item row="0" column="0" colspan="2">
     <widget class="QTextEdit" name="conversation"/>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="text_input">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="send_button">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
